src/parcsr_ls/par_amgdd_helpers.c

```diff
@@ -2164,7 +2164,7 @@ hypre_BoomerAMGDD_PackSendBuffer( hypre_ParAMGDDData *amgdd_data,
          total_num_nodes = hypre_AMGDDCompGridNumOwnedNodes(compGrid[level]) +
                            hypre_AMGDDCompGridNumNonOwnedNodes(compGrid[level]);
 
-         hypre_Memset(add_flag[level], 0, sizeof(HYPRE_Int)*total_num_nodes, HYPRE_MEMORY_HOST);
+         hypre_Memset(add_flag[level], 0, sizeof(HYPRE_Int)*total_num_nodes, memory_location);
```

Comment:
@waynemitchell Does it look correct to you? Thanks

Reply:
Yes, I looked at this. I had allocated `add_flag` at `memory_location` above (even though this all only has host implementation right now), so this makes things consistent.